<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>
     background-color: #f5f5f5;
     font-family: Arial, sans-serif;
    </string>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>350</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string>
      color: #333;
      border-bottom: 2px solid #007BFF;
    </string>
    </property>
    <property name="text">
     <string>Chat Room</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="userListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>120</width>
      <height>410</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>
      border: 1px solid #aaa;
      border-radius: 5px;
      background-color: #fff;
     </string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>350</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>
      border: 1px solid #aaa;
      border-radius: 5px;
      background-color: #fff;
      padding: 5px;
     </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>290</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>
      border: 1px solid #aaa;
      border-radius: 5px;
      background-color: #fff;
      padding: 5px;
     </string>
    </property>
    <property name="placeholderText">
     <string>Type your message...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>430</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>
      background-color: #007BFF;
      color: #fff;
      border: none;
      border-radius: 5px;
      padding: 10px;
     </string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Search </string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>

